app/templates/_footer_categories.html

```diff
@@ -5,7 +5,7 @@ <h2>
     </h2>
     <ul>
       <li>
-        <a href="mailto:enquiries@digitalmarketplace.service.gov.uk">Digital Marketplace support</a>
+        <a href="/support">Digital Marketplace support</a>
```

Comment:
This link is going to need to be changed individually in all the other apps too I think 😞 

Ideally I think we want a `url_for('.support')` rather than a hard-coded path here, and an external route defined here: https://github.com/alphagov/digitalmarketplace-utils/blob/master/dmutils/external.py so that other apps can use `url_for` to link to the page in their footers.